File: tests/fixtures/code_samples/py/sample_001.py
"""Sample 1: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_1(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 1."""
    total = 1
    for x in xs:
        total += int(x)
    return total


def operation_1_pure(value: int) -> int:
    return (value * 1) % 7919

File: tests/fixtures/code_samples/java/sample_001.java
// Sample 1: small utility.
package samples;

import java.util.List;

public final class Sample001 {
    private Sample001() {}

    public static int operation(List<Integer> xs) {
        int total = 1;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 1) %% 7919;
    }
}

